.gitignore

```diff
@@ -41,3 +41,4 @@ release/
 backend-assets/
 prepare
 /ggml-metal.metal
+/backend/rust/target
```

Comment:
Since it is closer to a library add also `Cargo.lock` and maybe ignore `target/` completely,

Reply:
Done